### ISSUE TITLE
Incorrect error; really "Sorry, you have exceeded your sharing quota"?

### ISSUE THREAD

**Issue by DeeperWaters** [ericyd/gdrive-transfer#1]
**tl;dr** I'm hitting the "Sorry, you have exceeded your sharing quota" error on the GDrive web GUI, and in your app log I am seeing "Error, you aren't the owner of this file/folder" for items I do own. Suspect it's a quota issue - any way around it?

Beautiful app!

With thousands of files/folders though, after it worked well for 20 or so transfers, I see the status in your log "Error, you aren't the owner of this file/folder" for items which I do own (and are native folders/files). 

Any way around this, hoping that the API has higher limits than the GUI? FWIW it still works through API Explorer (one painstaking fileID at a time).

I had suspected it could be overcome by setting sendNotificationEmails to false, but you have some form of that in transferFile.js line 25. [Here's a stackoverflow thread on it re API usage,](https://stackoverflow.com/questions/29630595/google-drive-api-permissions-rate-limit-exceeded-user-message-sorry-you-h) and [details on Permissions including sendNotificationEmails](https://developers.google.com/drive/v2/reference/permissions/insert) BUT the email still fires for ownership change.

> Whether to send notification emails when sharing to users or groups. This parameter is ignored and an email is sent if the role is owner. (Default: true)

**Comment by ericyd** [comment]
Yeah, this app is definitely buggy. It basically shares a codebase with [gdrive-copy](https://github.com/ericyd/gdrive-copy) which you can see has lots of [open issues](https://github.com/ericyd/gdrive-copy).

I'm not inclined to make serious updates to this app at this point because I don't think this is a simple update to make and I have no idea where to start.

I will gladly review any pull requests.

**Comment by DeeperWaters** [comment]
Thanks Eric; I understand. For a different folder I used gdrive-copy, and it worked flawlessly. That method doesn't generate emails (since the API respects sendNotificationEmails), and thus doesn't hit the quota limit.

This method has the advantage of preserving shared links (may help if many were externally shared), and AFAIK works perfectly for up to 50 items/day. So just spitballing here: if you do advance it, I think (from various boards) that the limit is 50 sharing events / day, each of which sends an email to the new owner. And you can share up to 50 items per event. So if you _batch_ 50 items before doing the send, you might increase it from 50 items/day to 2,500 items/day.  

Great work. I've asked how to send a donation!
